### PR TITLE
Update: Improve a guide about how to use executeTx and ExecParams

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

File: docs/guide/execute-transaction.md [hunk 1]
```diff
@@ -4,8 +4,7 @@ layout: splash
 
 # Execute Transaction
 
-`executeTx` is a high level method of Deployer which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/App, Opt-In, Transfers, Delete, Destroy etc. `executeTx` takes `ExecParams[]` as parameter.
-If the length of `ExecParams` array is greater than one, it will be considered as `atomic transaction`.
+`executeTx` is a high level method of Deployer which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/App, Opt-In, Transfers, Delete, Destroy etc. `executeTx` takes `ExecParams[]` as parameter. If the length of `ExecParams` array is greater than one, it will be considered as `atomic transaction` otherwise `single transaction`.
```
Comment (sczembor) [inline docs/guide/execute-transaction.md:7]: it also takes `SignedTransaction`

File: docs/guide/execute-transaction.md [hunk 2]
```diff
@@ -15,6 +14,35 @@ Note: For each parameter `type` and `sign` attributes are mandatory.
 
 Depending on the transaction `type`, other attributes will specify a signer of the transaction. For example, for `TransactionType.TransferAlgo` the `fromAccount` must be a full account (with secret key) and will sign the transaction.
 
+`deployer.executeTx` returns list of `TxnReceipt`, which extends `ConfirmedTxInfo`.
```
Comment (sczembor) [inline docs/guide/execute-transaction.md:17]: lets also add example with `signedTransaction`

File: docs/guide/execute-transaction.md (hunk 1 above)
Comment (rmeena840) [reply docs/guide/execute-transaction.md]: Ok. Done.

File: docs/guide/execute-transaction.md [hunk 3]
```diff
@@ -15,6 +14,43 @@ Note: For each parameter `type` and `sign` attributes are mandatory.
...
+
+```js
+const receipts = deployer.executeTx([txn0, txn1]);
+console.log("txn0 information: ", receipts[0]);
+console.log("txn1 information: ", receipts[2]);
+```
+
+Below examples demonstrates, how to perform application call using `ExecParam` and `WebMode`.
```
Comment (robert-zaremba) [inline docs/guide/execute-transaction.md:25]:
```suggestion
Below we demonstrate how to perform application call using `ExecParam` and `WebMode`:
```

File: docs/guide/execute-transaction.md [hunk 4]
```diff
@@ -15,6 +14,43 @@ Note: For each parameter `type` and `sign` attributes are mandatory.
...
+				},
+				appID: 100,
+				payFlags: {},
+			},
+		];
+		webMode.executeTx(tx);
+	};
+```
+
+Below examples demonstrates, how to use `SignedTransaction` and `executeTx` in runtime.
```
Comment (robert-zaremba) [inline docs/guide/execute-transaction.md:46]:
```suggestion
We can also use `SignedTransaction` and `executeTx` in runtime:
```

**Review by robert-zaremba — COMMENTED** [review]
Added few text suggestions.
@sczembor please make sure that the guide is exhaustive and contains all information required for a new developer.

File: docs/guide/execute-transaction.md [hunk 5]
```diff
@@ -15,6 +14,43 @@ Note: For each parameter `type` and `sign` attributes are mandatory.
...
+				},
+				appID: 100,
+				payFlags: {},
+			},
+		];
+		webMode.executeTx(tx);
+	};
+```
+
+We can also use `SignedTransaction` and `executeTx` in runtime:
```
Comment (sczembor) [inline docs/guide/execute-transaction.md:46]: we could add link to an example from Algorand docs with the signedTransaction. How it is created using `algosdk`. Or just add it to our example

File: docs/guide/execute-transaction.md [hunk 6]
```diff
@@ -4,8 +4,7 @@ layout: splash
 
 # Execute Transaction
 
-`executeTx` is a high level method of Deployer which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/App, Opt-In, Transfers, Delete, Destroy etc. `executeTx` takes `ExecParams[]` as parameter.
-If the length of `ExecParams` array is greater than one, it will be considered as `atomic transaction`.
+`executeTx` is a high level method of Deployer which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/App, Opt-In, Transfers, Delete, Destroy etc. `executeTx` takes `ExecParams[]` or `SignedTransaction[]` as parameter. If the length of `ExecParams` array is greater than one, it will be considered as `atomic transaction` otherwise `single transaction`.
```
Comment (sczembor) [inline docs/guide/execute-transaction.md:7]: Lets add a section where we shortly describe `ExecParams` and `SignedTransaction`. The differences and why would one prefer to use one over another

Comment (rmeena840) [reply docs/guide/execute-transaction.md]: I have created a section for ExecParams and we already have a section for SignedTransaction. I have also highlighted the difference between them.

**Comment by rmeena840** [comment]
@sczembor Ready for another review.

File: docs/guide/execute-transaction.md (hunk 5 above)
Comment (rmeena840) [reply docs/guide/execute-transaction.md:46]: We already have signed transaction example at the end of this doc.

**Review by sczembor — APPROVED** [review]
Thanks for the PR

**Pull request merged**